frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -519,7 +519,7 @@ class Worksheet extends React.Component {
         } else {
             // If no location for the insertion is specified,
             // insert the new item at the top of the worksheet in default.
-            actualData['after_sort_key'] = -1;
+            actualData['after_sort_key'] = 0;
```

Comment:
To solve issues like this in the future, maybe we should define a constant called `WORKSHEET_TOP_POS` and set it to -1, and use it in all places that depend on the worksheet top position. What do you think? (we could probably do this in another PR as this is just a general refactoring thing though)

Reply:
Yes, I think it is a good suggestion!

Reply:
Tracking in https://github.com/codalab/codalab-worksheets/issues/3623